core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -629,7 +629,7 @@ oppia.factory('explorationPropertyService', [
       // Validates the given value and returns a boolean stating whether it
       // is valid or not. THIS CAN BE OVERRIDDEN BY SUBCLASSES. The default
       // behavior is to always return true.
-      _isValid: function(value) {
+      _isValid: function() {
```

Comment:
This file has been written long before, I have just refactored a function(not this one) and the above change is done by my linter. IMO we should have a separate PR to apply the new naming convention to this and all other files which have old names.